eng/common/pipelines/templates/steps/docindex.yml

```diff
@@ -49,6 +47,7 @@ jobs:
           targetPath: $(Build.ArtifactStagingDirectory)/docfx_project/_site
           
       - pwsh: |
+          git -c user.name=`"azure-sdk`" -c user.email=`"azuresdk@microsoft.com`" commit -am "Save the language updates."
```

Comment:
I don't think the backticks are necessary?

Reply:
Yeah, @weshaggard suggested to revert the changes instead of committing the changes. Will change it here.